### PR TITLE
Docs: Updated what's new article

### PR DESCRIPTION
**Pull request by torkelo**
Updated what's new article for v6.6

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -4,6 +4,9 @@ docs:
 	docker pull grafana/docs-base:latest
 	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make webpack && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
 
+docs-local:
+	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make webpack && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
```
Comment (torkelo) [inline docs/Makefile]: added this so you can run docs with locally built grafana/docs-base to see your recently added images

**Comment by torkelo** [comment]
@robbymilo I could not get the menu.yaml to work, it has the Version 6.6 in the menu but I could not get it to show. I also added Reporting to features section (a big problem with switch to new docs was that this was missed in the menu), but I added it to features in menu yaml file, restarted the docs but it does not show in menu still. Any ideas?

File: docs/Makefile (hunk 1 above)
Comment (marefr) [reply docs/Makefile]: I don't see any difference with this compared to `docs` target? Maybe I'm tired?

Comment (torkelo) [reply docs/Makefile]: The docs target downloads the latest from docker hub, or does it use local for you if local is newer?

Comment (torkelo) [reply docs/Makefile]: I tried again, and if I use `make docs` that do the docker pull before I loose all my newly added images that are not in the pushed docker image yet

File: docs/sources/guides/whats-new-in-v6-6.md [hunk 2]
```diff
@@ -34,12 +34,71 @@ Grafana 6.6 comes with a lot of new features and enhancements:
...
+- Calendar automatically shows when from or to inputs has focus
+- A single calendar view can be used to select and show the from & to date.
+- Select recent absolute ranges
+
+{{< docs-imagebox img="/img/docs/v66/time_picker_update.png" max-width="700px" caption="New time picker" >}}
+
+## News panel
+
+This panel supports showing RSS feeds as news items. It is used in the updated default home dashboard. Add it to
+your custom home dashboards to keep up to date with Grafana news or switch the default RSS feed to one of your choice.
```
Comment (dprokop) [inline docs/sources/guides/whats-new-in-v6-6.md]: Should we add info that feed needs to be a RSS feed? Tried Atom feed and it didn't work https://www.theverge.com/rss/index.xml

**Pull request merged**